docs/BinSkimRules.md

```diff
@@ -914,3 +914,25 @@ This rule emits CSV data to the console for every compiler/language/version comb
 
 ---
 
+## Rule `BA6001.DisableIncrementalLinking`
```

Comment:
>DisableIncrementalLinking [](http://example.com/codeflow?start=16&length=25)

I like the balance you appear to be striking in analysis. If you see evidence that suggests the standard is relevant (i.e., an actual release build), you fault-in the analysis, otherwise we drop it.

Reply:
Thank you :).  Yes, that is the goal.  I'm trying to make the analysis as valuable as possible by reducing false positives or cases where it warns for things that don't matter.

Reply:
For performance, people seem generally not be overly upset about false negatives. The rate of discovery for perf issues related to compilation settings is low and the psychology of the issues is different than for security (where people rightly can get agitated over a missed finding). I think firing warnings is a good idea, then we can enable these by default and interleave them into regular analysis. Teams that want to be strict about certain settings can use a config file to elevate to error.